cmd/object-api-interface.go

```diff
@@ -26,10 +26,14 @@ import (
 	"github.com/minio/minio/pkg/policy"
 )
 
+// CheckCopyPreconditionFn returns true if copy precondition check failed.
```

Comment:
@aead, the precondition check tests more than one header - such as ETag and if-modified-since. If it is written as above, error check will need to be redone to write the correct error headers at the handler.

Reply:
That cleanup we might have to do later @aead